integration/content/client-worker/worker.js

```diff
@@ -0,0 +1,14 @@
+import * as pmrpc from '../../../src/pm-rpc/index'
+
+onmessage = function (event) {
+    if (event.data.message === 'init') {
+        pmrpc.api.request('api-for-worker')
+            .then(api => {
+                if (api.add) {
+                    api.add(2, 3).then(result => {
+                        postMessage({message: 'api received', result})
+                    })
+                }
```

Comment:
if `api.add` does not exists, your test will hang until timeout,
please send `postMessage({message: 'test failed' })` if api is not correct
and in the test itself, if the message was `test failed` call `done.fail()`